core/test_integration.py

```diff
@@ -47,7 +47,6 @@ def tearDownClass(cls):
         super().tearDownClass()
 
         if getattr(cls, 'browser') is not None:
-            time.sleep(2)  # Keep the browser open for 2 seconds
             cls.browser.quit()
```

Comment:
Got rid of implicit wait. It was a debugging thing I used when running the browser.. non-headless? head? lol